.github/dependabot.yml

```diff
@@ -3,4 +3,38 @@ updates:
 - package-ecosystem: pip
   directory: results-processor/
   schedule:
-    interval: monthly
+    interval: weekly
+- package-ecosystem: gomod
+  directory: /
```

Comment:
[Most examples](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#example-dependabotyml-file) I see add "" around the value, e.g. "/", "weekly"

Reply:
Done!